pallets/parachain-staking/Cargo.toml

```diff
@@ -6,18 +6,18 @@ name = "parachain-staking"
 version = "1.6.2"
 
 [dev-dependencies]
-pallet-aura = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false}
-pallet-timestamp = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false}
-sp-consensus-aura = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false}
-sp-core = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false}
-sp-io = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false}
+pallet-aura = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19"}
```

Comment:
`default-features = false` is not needed and should not be used in `dev-dependencies`